TD Balance Changes 12242016.

AoAGeneral opened the pull request:
Add detection to temple of Nod.
(Nice to have detection then stealthers sneaking by it)

artillery minimum range increase from 2c896 to 3c0.

mammoth tank HP increase from 800 to 870.
(Dies by medium tank spams and e3 a little to easily from recent changes. With a longer build time they fall prey to mass numbers.)

ion time increase from 3:00 to 4:30.
(Ion strikes go off a little to fast. Mammoth armies take about 2:00 to build back up. By the time base attacks happen close to 3:00)

nuke timer from 5 to 6.
(Because the nuke damage has been buffed)

nuke damage spread from 2c512 to 5c512.
(Nuke damage spread was to weak. Barely killing a power plant where it lands just a few cells away from base hit)

nuke spread damage increased from 100 to 110. (First spread modifier)
(See line above)

a10 timer increased from 3:30 to 4:00.
(Timer increased because they were going off a little to fast still. These planes kill infantry builds as it is.)

MSAM HP from 120 to 180.
(To weak vs air. Missiles drain takes a few seconds to reload and then sniped within that time. Specially since dedicated AA)

Commando weapon range increased from 6c0 to 8c0.
(They had the same range as e3 and minigunners were able to close distances on them extremely fast. Take note they shoot farther then their vision range.)

Commando weapon projectile speed increased from 1c682 to 5c682.
(This was super annoying. You fire a shot and the infantry doesn't die because bullet travel to slow. Increased this fixed the problem.)

Commando price increase from 1000 to 2000.
(Extremely cheap when buffed. Increased price to match their power.)

Commando build time increased from 24 to 32. (2000 Build duration)
(Build time a little to fast. Compensates this with multiple barracks.)

Husks interval timer increased from 2 to 6.
(Husks blow up to fast to capture anything. Kept on low timer to prevent vehicle block. Only possible way is to have an engineer in an APC ready. Which Nod is unable to have APCs.)

Bike damage increase from 30 to 31.
(HP changes on harvs, structures, and mammoth. A very small change leading to a decent buff. Granting a 2 damage increase with constant hits)

Chem damage vs none reduced from 100 to 70.
(Chems performing a little to well by themselves. Having mixed units is a better solution. IE: My recent flame tank chem trooper mix)

Chem damage vs armor increased from 50 to 75.
(Chems performed a little to weak vs armor. Doing less damage then infantry and getting squashed by medium tanks and light tanks. This will make players think twice about squashing tactics. Also more damage vs armored defenses.)

Grenadier accuracy reduced from 213 to 813. (Makes them miss a lot more).
(Grenadiers were to accurate and unable to kill infantry packs. Looking at CNC95 and RA in ORA they miss their targets as well. Having them miss spreads their AoE damage and able to take out infantry packs. Players maye want to think of spreading infantry when fighting grenadiers)

Flamer damage vs armor reduced from 20 to 10. (Flame infantry)
(To much damage to APCs and light tanks)

Flamer damage vs none increased from 100 to 110. (Flame infantry.)
(Made them stronger vs infantry to balance out vs chems. Makes them a strong viable choice.)

Harv build timer increase from 24 to 27. (1680 build timer)
(Harv spamming is like a disease. Its gotten to strong to quickly. Increasing their timer slightly also increases other viable builds.)

Harvester HP increase from 600 to 625.
(Both medium tanks and bikes doing a little to much damage. Compensates slightly for build timer)

Refinery power reduced from 50 to 40.
(Offers more build choices. Opening barracks becomes stronger then going strip/factory all the time. Both openings strong strats just requires scouting.)

Refinery sell no longer refunds.
(Includes with the harv spamming. Gave 250 back which counted towards buggy/bike money. IE: PP > Ref > Strip (Strip makes 2-3 buggies 2 bikes then a harv) > Ref (Sell ref gain harv) > Strip (Because power only 30) > PP

Refinery tiberium hold reduced from 2000 to 700.
(This has had no impact to force people to build silos. But it opts people to protect refineries rather then "Lost a ref. Meh. Got 3 others.". Losing a ref now loses earned money. If it holds money. Also prevents sell trick to save power.)

Silo hold increase from 2000 to 3000.
(Compensate for Refinery. Also has extra HP now. See below.)

AGT HP from 600 to 550.
(Way to strong when it came to units coming to these in mass amounts. Also hard for artillery units to shell due to small box size and HP pool.)

Light Tank price reduced from 700 to 650.
(700 was a bit to high. 650 seems to hit the right spot but left build time as is. 0:17)

Visceroid spawn reduce from 10 to 5.
(This was a weird one. Infantry dieing in tiberium was at a good spot. But for some reason when chems kill infantry it would create 2-3 visceroids. Then those visceroids would create 2-3 at a time. RNG issue so was reduced.)

Increase all building HP by 100.
(Help balance vs medium tanks, bikes, light tanks, chems, grenadiers, air units, and stealth tanks.)

Gun turret HP increase from 400 to 410.
(This is taking baby steps. I want them a little tougher vs medium tanks but not at the point of RA style. Same logic as obrakmann taking baby steps with MCV.)

Starting units adjusted.
(This created a big problem in competitive scenes. Having early units is good to prevent infantry rushes. But starting with a bike/apc while your opponent starts with a hummer/buggy is to off balancing.)

MCV build time decreased from 1:36 to 1:00. (3750)
(Price unchanged. Takes a little to long in team games and some 1v1 scenarios to get one built. Also now drains money faster to compensate)

Engineer HP increase from 25 to 30.
(Dies a little to quick hummers/buggies. Same kill time as minigunner (If takes a hit while standing)

APC HP increase from 210 to 215.
(These are dieing a little to fast and weak vs certian units. IE: buggies/hummers and bikes. Tested vs light tanks and more optimal. Baby steps.)

APC Damage vs light decreased from 105 to 100.
(I don't want them killing bikes to fast now that they are cheaper.)

APC cost decreased from 600 to 550.
(APCs for infantry builds as GDI was waaaaay to expensive. Having a slightly faster build timer of 0:14 instead of 0:15 will help too. Helps to withstand early onslaught of Nod. Keep note bikes still 0:12 timer)

(Created new Sam weapon Dragon from RA e3 AA. Reason: Shared weapon with AGT.)

Samsite range increased from 8c0 to 12c0.
(Dedicated AA. Keep note: Longer range = longer time to kill a unit and able to escape. Closer it is the faster air units die.)

Samsite spread damage decreased from 682 to 128.
(To much AoE damage)

Samsite range limit increased from 9c614 to 15co.
(Missiles would die out within its own range circle.)

Samsite reveal shroud decreased from 8c0 to 6c0.
(Prevents the RA logic of build an AA in front of your base and all good. You need units/structures to make good use of its range.)

Samsite now fires two shots at a time.
(Increases damage and intuitive to home base of CNC95.)

Recon Bike HP decreased from 120 to 110.
(Takes to long to kill these units when vs minigunners and light vehicle types. APCs taking to long to kill these.)

Airstrip/Weapon Factory power increased from 30 to 50.
(Increased like a disease. Nod players going with an extremely fast 4 strip build without com center. GDI maxing at 3 factories with non stop vehicles. Infantry became to weak.)

Communication Center power increased from 40 to 50.
(See above.)

APC movement speed increased from 128 to 132.
(A little to slow vs light tanks and dealing with buggies. Buggies being faster still they are able to get a few extra shots off. Provides better base entry.)

MRLS range reduced from 12c0 to 11c0.
(Out ranging Nod artillery. Extremely accurate as is and to strong vs several units. Shorter range allows faster vehicles to reach them better.)

MRLS damage vs none reduced from 35 to 25.
(To much damage vs infantry. Specially to e3.)

Obelisk power reduced from 150 to 90.
(Having to build an adv power plant after each Obelisk is put up is insane. Two Obelisk = 180 now and adv power plant is 200.)

Website reference at http://www.sleipnirstuff.com/forum/viewtopic.php?f=82&p=299238#299238

AoAGeneral committed:
TD Balance Changes 12242016.

Add detection to temple of Nod.
(Nice to have detection then stealthers sneaking by it)

artillery minimum range increase from 2c896 to 3c0.

mammoth tank HP increase from 800 to 870.
(Dies by medium tank spams and e3 a little to easily from recent changes. With a longer build time they fall prey to mass numbers.)

ion time increase from 3:00 to 4:30.
(Ion strikes go off a little to fast. Mammoth armies take about 2:00 to build back up. By the time base attacks happen close to 3:00)

nuke timer from 5 to 6.
(Because the nuke damage has been buffed)

nuke damage spread from 2c512 to 5c512.
(Nuke damage spread was to weak. Barely killing a power plant where it lands just a few cells away from base hit)

nuke spread damage increased from 100 to 110. (First spread modifier)
(See line above)

a10 timer increased from 3:30 to 4:00.
(Timer increased because they were going off a little to fast still. These planes kill infantry builds as it is.)

MSAM HP from 120 to 180.
(To weak vs air. Missiles drain takes a few seconds to reload and then sniped within that time. Specially since dedicated AA)

Commando weapon range increased from 6c0 to 8c0.
(They had the same range as e3 and minigunners were able to close distances on them extremely fast. Take note they shoot farther then their vision range.)

Commando weapon projectile speed increased from 1c682 to 5c682.
(This was super annoying. You fire a shot and the infantry doesn't die because bullet travel to slow. Increased this fixed the problem.)

Commando price increase from 1000 to 2000.
(Extremely cheap when buffed. Increased price to match their power.)

Commando build time increased from 24 to 32. (2000 Build duration)
(Build time a little to fast. Compensates this with multiple barracks.)

Husks interval timer increased from 2 to 6.
(Husks blow up to fast to capture anything. Kept on low timer to prevent vehicle block. Only possible way is to have an engineer in an APC ready. Which Nod is unable to have APCs.)

Bike damage increase from 30 to 31.
(HP changes on harvs, structures, and mammoth. A very small change leading to a decent buff. Granting a 2 damage increase with constant hits)

Chem damage vs none reduced from 100 to 70.
(Chems performing a little to well by themselves. Having mixed units is a better solution. IE: My recent flame tank chem trooper mix)

Chem damage vs armor increased from 50 to 75.
(Chems performed a little to weak vs armor. Doing less damage then infantry and getting squashed by medium tanks and light tanks. This will make players think twice about squashing tactics. Also more damage vs armored defenses.)

Grenadier accuracy reduced from 213 to 813. (Makes them miss a lot more).
(Grenadiers were to accurate and unable to kill infantry packs. Looking at CNC95 and RA in ORA they miss their targets as well. Having them miss spreads their AoE damage and able to take out infantry packs. Players maye want to think of spreading infantry when fighting grenadiers)

Flamer damage vs armor reduced from 20 to 10. (Flame infantry)
(To much damage to APCs and light tanks)

Flamer damage vs none increased from 100 to 110. (Flame infantry.)
(Made them stronger vs infantry to balance out vs chems. Makes them a strong viable choice.)

Harv build timer increase from 24 to 27. (1680 build timer)
(Harv spamming is like a disease. Its gotten to strong to quickly. Increasing their timer slightly also increases other viable builds.)

Harvester HP increase from 600 to 625.
(Both medium tanks and bikes doing a little to much damage. Compensates slightly for build timer)

Refinery power reduced from 50 to 40.
(Offers more build choices. Opening barracks becomes stronger then going strip/factory all the time. Both openings strong strats just requires scouting.)

Refinery sell no longer refunds.
(Includes with the harv spamming. Gave 250 back which counted towards buggy/bike money. IE: PP > Ref > Strip (Strip makes 2-3 buggies 2 bikes then a harv) > Ref (Sell ref gain harv) > Strip (Because power only 30) > PP

Refinery tiberium hold reduced from 2000 to 700.
(This has had no impact to force people to build silos. But it opts people to protect refineries rather then "Lost a ref. Meh. Got 3 others.". Losing a ref now loses earned money. If it holds money. Also prevents sell trick to save power.)

Silo hold increase from 2000 to 3000.
(Compensate for Refinery. Also has extra HP now. See below.)

AGT HP from 600 to 550.
(Way to strong when it came to units coming to these in mass amounts. Also hard for artillery units to shell due to small box size and HP pool.)

Light Tank price reduced from 700 to 650.
(700 was a bit to high. 650 seems to hit the right spot but left build time as is. 0:17)

Visceroid spawn reduce from 10 to 5.
(This was a weird one. Infantry dieing in tiberium was at a good spot. But for some reason when chems kill infantry it would create 2-3 visceroids. Then those visceroids would create 2-3 at a time. RNG issue so was reduced.)

Increase all building HP by 100.
(Help balance vs medium tanks, bikes, light tanks, chems, grenadiers, air units, and stealth tanks.)

Gun turret HP increase from 400 to 410.
(This is taking baby steps. I want them a little tougher vs medium tanks but not at the point of RA style. Same logic as obrakmann taking baby steps with MCV.)

Starting units adjusted.
(This created a big problem in competitive scenes. Having early units is good to prevent infantry rushes. But starting with a bike/apc while your opponent starts with a hummer/buggy is to off balancing.)

MCV build time decreased from 1:36 to 1:00. (3750)
(Price unchanged. Takes a little to long in team games and some 1v1 scenarios to get one built. Also now drains money faster to compensate)

Engineer HP increase from 25 to 30.
(Dies a little to quick hummers/buggies. Same kill time as minigunner (If takes a hit while standing)

Engineers now capture externally. (RA Mechanic)
(This was put in to allow a timer for capturing. I never really liked having two engineers to capture and having one by itself to instantly capture was bleh. Also helps vs 50% issue)

Engineers now capture structures in 5 seconds.
(This allows to compensate for the 50% damage modifier to capture and gives enemies a chance to kill.)

Engineer is consumed upon capture.
(I don't like RA style of one engineer capturing a map. Invest in the 500$.)

APC HP increase from 210 to 215.
(These are dieing a little to fast and weak vs certian units. IE: buggies/hummers and bikes. Tested vs light tanks and more optimal. Baby steps.)

APC Damage vs light decreased from 105 to 100.
(I don't want them killing bikes to fast now that they are cheaper.)

APC cost decreased from 600 to 550.
(APCs for infantry builds as GDI was waaaaay to expensive. Having a slightly faster build timer of 0:14 instead of 0:15 will help too. Helps to withstand early onslaught of Nod. Keep note bikes still 0:12 timer)

(Created new Sam weapon Dragon from RA e3 AA. Reason: Shared weapon with AGT.)

Samsite range increased from 8c0 to 12c0.
(Dedicated AA. Keep note: Longer range = longer time to kill a unit and able to escape. Closer it is the faster air units die.)

Samsite spread damage decreased from 682 to 128.
(To much AoE damage)

Samsite range limit increased from 9c614 to 15co.
(Missiles would die out within its own range circle.)

Samsite reveal shroud decreased from 8c0 to 6c0.
(Prevents the RA logic of build an AA in front of your base and all good. You need units/structures to make good use of its range.)

Samsite now fires two shots at a time.
(Increases damage and intuitive to home base of CNC95.)

Recon Bike HP decreased from 120 to 110.
(Takes to long to kill these units when vs minigunners and light vehicle types. APCs taking to long to kill these.)

Airstrip/Weapon Factory power increased from 30 to 50.
(Increased like a disease. Nod players going with an extremely fast 4 strip build without com center. GDI maxing at 3 factories with non stop vehicles. Infantry became to weak.)

Communication Center power increased from 40 to 50.
(See above.)

APC movement speed increased from 128 to 132.
(A little to slow vs light tanks and dealing with buggies. Buggies being faster still they are able to get a few extra shots off. Provides better base entry.)

MRLS range reduced from 12c0 to 11c0.
(Out ranging Nod artillery. Extremely accurate as is and to strong vs several units. Shorter range allows faster vehicles to reach them better.)

MRLS damage vs none reduced from 35 to 25.
(To much damage vs infantry. Specially to e3.)

Obelisk power reduced from 150 to 90.
(150 is the same power as Temple of Nod. Having to build an adv power plant every single time you want more then one Obelisk is insane.)

Website reference at http://www.sleipnirstuff.com/forum/viewtopic.php?f=82&p=299238#299238

pchote commented:
> Engineers now capture structures in 10 seconds.

This seems excessively long.  The long capture time in RA makes engineers useless for anything but tech structures, and we don't want TD to go the same way.

AoAGeneral commented:
I agree actually. Comparing to RA. Originally when I tested five seconds it seemed fast. Perhaps seven seconds is the best method.

In RA I do agree it takes way to long. (15 seconds)

pchote commented:
The engineers are consumed at $500 a pop, so fast isn't necessarily bad – it should be only *possible* to defend against engineers, not *guaranteed*.  Its also worth consider that units die quicker in TD than RA, so you can multiply whatever value you pick for TD by probably 1.5 - 2 when comparing with RA.

AoAGeneral commented:
This is true. While taking into consideration that TD has faster moving units then RA as well. Ive updated with 7 seconds.

pchote commented:
It takes approximately 2.5 seconds for a minigunner to kill an engineer, which means that even at 7 seconds a single $100 minigunner could singlehandedly hold off nearly three $500 engineers.  Do you really think that's fair?

AoAGeneral commented:
Is this 2.5 seconds damage vs 25 HP or 30 HP?

pchote commented:
Good catch, however 30 HP is still only 4 bursts, or ~2.9 seconds.

AoAGeneral commented:
Im going to go ahead and put it on 5 seconds. However, if they prone they can take additional damage which would be 5 shots. (Comparing to a minigunner). This reverts back to the prone mechanic idea here https://github.com/OpenRA/OpenRA/issues/10967 which gives infantry extra armor.

As mentioned though I will change it to 5.

GraionDilach commented:
Wait, this is stupid. Okay, I can sorta relate with the reasoning behind externalcapturing buildings in TD - it's meh though - but externalcapturing husks with the Engineer is stupid and I will :-1: this PR as-is due to that. Huskcapturing is already situational, this will render it useless.

obrakmann commented:
> MCV build time decreased from 1:36 to 1:00. (3750)
> 
> MCV no longer needs communication center. (Only repair pad)

What's the reasoning behind this change?

Also someone needs to check if any of these changes break the campaign missions.

pchote commented:
> externalcapturing husks with the Engineer is stupid

Husks and tech buildings still have the normal capturing traits right now, so they won't be capturable at all now (this needs to be fixed!).  It would be better if the engineer could keep the regular `Captures` trait for internally capturing husks.

AoAGeneral commented:
@obrakmann You cant build MCVs in the campaign missions until the very last level of the missions itself (Exceptions being Covert Op missions.)

They were also an extremely long build time compared to RA in its original form as example. The ability to make MCVs via repair pad only allows for more optional and risky build orders. The ref power at 40 from 50 is one example.

@GraionDilach I do agree that they should be able to capture husks but every vehicle in TD does leave a husk behind. RA is different where rangers and APCs do not leave husks unlike TD.

If the husk capturing is external/internal it is still useful. Just because of either or does not mean its useless. Currently right now no one captures husks because of the short timer before they burn out. So unless there is a trait I am missing I can work on fixing this.

pchote commented:
I agree with @obrakmann that the MCV change looks to be too extreme to predict the consequences.  Can we build up to that over a couple of releases, if it turns out to be necessary?

pchote commented:
The changes here are focusing on a few different actors and look to have distinct motivations.  Can you please edit into the initial PR description a sentence or two explaining the motivations for each set of changes?

AoAGeneral commented:
@obrakmann Changed the MCV back. I still think it provided an additional build. It takes 1:00 to build a MCV and costing at 4000 is nearly your spending at the start of the game. Following the logic course though and taking the smaller steps.

@pchote edited the info above.

reaperrr commented:
The lint/Travis error message is bogus, but there actually is an issue with gdi03 regardless: in its rules.yaml it negates `Capturable` on the SAM Site, where it needs to negate `ExternalCapturable` now.

pchote commented:
> OpenRA.Utility(1,1): Error: Map contains invalid custom rules.

That error isn't bogus.  The second error happens because the custom map rules can't be loaded, and the map doesn't define any spawn point to use with the default rules.

AoAGeneral commented:
@pchote @reaperrr 

The engineer capture logic may get put onto a future release. Im getting feedback in the forums of players not wishing for the engineer change. So this may get fixed for now anyways. (Please see linked thread post in PR comments)

What may end up happening is play a bunch of games and watch specifically for any engineer activities. If there is a large lack of them still I will put the subject up again. Currently at work though but when im home will revert changes.

pchote commented:
Some people don't like change.  I remember similar arguments against introducing the base radius, tanks crushing infantry, and even 5 infantry sharing a cell back in the day.  If you are confident that a change will be positive for gameplay then you shouldn't let the nay-sayers scuttle, it unless they can justify their objections with actual logic.

AoAGeneral commented:
@pchote 

I added in the Obelisk power change because I already know for (And from previous testing) that having to build an adv power plant each time you want more then one obelisk is an issue. Updated the comments.

I have no further plans to alter the Obelisk until after next release. I added in the power change since the Obelisk fix is on the next release milestone.

Obelisk power reduced from 150 to 90.
(Having to build an adv power plant after each Obelisk is put up is insane. Two Obelisk = 180 now and adv power plant is 200.)

GraionDilach commented:
This needs to be rebased again because Travis didn't run on it and IIRC it still breaks one of the missions.

AoAGeneral commented:
I just did a rebase on it but Travis CI isn't picking up still. O.o
